doc/changelog.md

```diff
@@ -502,13 +593,20 @@
 
 <h3>Internal changes</h3>
 
+* Adds `catalyst.from_plxpr.from_plxpr` for converting a PennyLane variant jaxpr into a
+  Catalyst variant jaxpr.
+  [(#837)](https://github.com/PennyLaneAI/catalyst/pull/837)
+
+* Catalyst is now compatible with Enzyme `v0.0.130`
+  [(#898)](https://github.com/PennyLaneAI/catalyst/pull/898)
+
 * When memrefs have no identity layout, memrefs copy operations are replaced by the linalg copy operation.
   It does not use a runtime function but instead lowers to scf and standard dialects. It also ensures
   a better compatibility with Enzyme.
   [(#917)](https://github.com/PennyLaneAI/catalyst/pull/917)
 
-* llvm O2 and Enzyme passes are only run when needed (gradients presents). Async execution of QNodes triggers now triggers a
-   Coroutine lowering pass.
+* llvm O2 and Enzyme passes are only run when needed (gradients presents). Async execution of QNodes
+  triggers now triggers a Coroutine lowering pass.
```

Comment:
Could be user facing in the sense that compilation time is significantly reduced when no gradients are used.
```suggestion
* LLVM's O2 optimization pipeline and Enzyme's AD transformations are now only run in the presence
  of gradients, significantly improving compilation times for programs without derivatives.
  Similarly, LLVM's coroutine lowering passes only run when `async_qnodes` is enabled in the QJIT decorator.
```

Reply:
Agree, I'll move this one into improvements.